components/providers/provider-hero-image.tsx

```
@@ -0,0 +1,244 @@
+'use client';
+
+import React, { useState, useCallback } from 'react';
+import Image from 'next/image';
+import { toast } from 'sonner';
+import { Button } from '@/components/ui/button';
+import { Card, CardContent } from '@/components/ui/card';
+import { Upload, X, Camera, Loader2 } from 'lucide-react';
+import { Label } from '@/components/ui/label';
+import { getFallbackImageUrl } from '@/lib/image-utils';
```

ESLint

[error] 'getFallbackImageUrl' is defined but never used.

(@typescript-eslint/no-unused-vars)